src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,79 @@
+package duke.command;
+import duke.DukeException;
+import duke.task.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * MarkCommand represents the user's actions of marking/unmarking tasks.
+ */
+public class MarkCommand extends Command {
+
+    // mark boolean is true indicates the mark command, false the unmark command
+    private boolean mark;
```

Comment:
```suggestion
    private boolean isMark;
```
Same as above, more intuitive naming for a Boolean variable. Perhaps there are more of such issues in other files?